### ISSUE TITLE
Height is expressed as a Rate

### ISSUE THREAD

**Issue by greg0ire** [mhor/php-mediainfo#67]
The absolute value for a rate is a float, and for a height, I would expect an integer rather than a float. Not sure if this can be fixed though.

**Comment by mhor** [comment]
Actually I cast ```absoluteValue``` to float because attribute ```display_aspect_ratio``` is  a float type.
Example:
```["display_aspect_ratio"]=>
        object(Mhor\MediaInfo\Attribute\Rate)#39 (2) {
          ["absoluteValue":"Mhor\MediaInfo\Attribute\Rate":private]=>
          float(2.353)
          ["textValue":"Mhor\MediaInfo\Attribute\Rate":private]=>
          string(6) "2.35:1"
        }
```
To solve ```Rate::absoluteValue``` type I proposed to:
- add a new attribute type ```Ratio``` with properties ```absoluteValue``` (float type) and ```textValue``` (string type)
- cast ```Rate::absoluteValue``` to int
- release a ```3.0.0``` due to BC break

**Comment by greg0ire** [comment]
Sounds good!

**Comment by mhor** [comment]
Thank you @greg0ire , #68 is now merged and realeased as [3.0.0](https://github.com/mhor/php-mediainfo/releases/tag/3.0.0)
